src/query/filter-terms.sparql

```diff
@@ -0,0 +1,18 @@
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX schema: <http://schema.org/>
+PREFIX cube: <https://cube.link/>
+PREFIX ssz: <https://ld.stadt-zuerich.ch/schema/>
+PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+CONSTRUCT {
+  <urn:ssz:terms> a hydra:Collection ; hydra:member ?term .
```

Comment:
I'm in the process of understanding the code here. 
What is the role of hydra:Collection and hydra:member here?

Reply:
Good question! Thanks.

The form does not run queries directly (although I now think that maybe there should be such an extension). Instead, it knows how to create resource URLs from templates. Those resources should be instances of `hydra:Collection` which can support paging and filtering. See https://forms.hypermedia.app/#/extensions/hydra?id=dependent-fields

Since we do not have a backend, the `CONSTRUCT` queries mimic the representation of a collection resource.

Reply:
So it's an intermediate structure to be interpreted by the UI? I see :)